_episodes/40.md

```diff
@@ -0,0 +1,481 @@
+the Iceberg connector and what has changed in Iceberg as well!
+
+### What is Apache Iceberg?
+
+[Iceberg](https://iceberg.apache.org/) is a next-generation table format that 
+defines a standard around the metadata used to map data to a SQL query engine. 
+It addresses a lot of the maintainability and reliability issues many engineers
+experienced with the way
+[Hive modeled SQL tables]({% post_url 2020-10-20-intro-to-hive-connector %}) 
+over big data files. 
+
+One common confusion to point out is that table format is not equivalent to file
+formats like ORC or Parquet. The table format is the layer that maintains 
+metadata mapping these files to the concept of a table and other common database
+abstractions.
+
+This episode assumes you have some basic knowledge of Trino and Iceberg already. If
+you are new to Iceberg or need a refresher, we recommend the two older episodes
+about Iceberg and Trino basics:
+
+* [14: Iceberg: March of the Trinos]({{site.url}}/episodes/14.html)
+* [15: Iceberg right ahead!]({{site.url}}/episodes/15.html)
+
+### Why Iceberg over other formats?
+
+There has been some great advancements to big data technologies that brought
+back some SQL and data warehouse capabilities. However, Hive and Hive-like table
+formats are still missing some capabilities due to limitations that Hive tables
+have, such as dropping and reintroducing stale data unintentionally. On top of
+that, Hive tables require a lot of knowledge of Hive internals. More recent
+formats aim to remain backwards compatible with Hive, but inadvertently
+reintroduce these limitations.
+
+On top of that, Iceberg has the most support for query engines and puts a heavy 
+emphasis on being a format that is interoperable. This improves the level of
+flexibility users have to address a wider array of use cases that may involve
+querying over a system like Snowflake or a data lakehouse running with
+Iceberg. All of this is made possible by the
+[Iceberg specification](https://iceberg.apache.org/spec) that all these query
+engines must follow.
+
+Finally, a great video presented by Ryan Blue that dives into Iceberg is, 
+"[Why you shouldn't care about Iceberg](https://www.youtube.com/watch?v=_GW3GYZK66U)."
+
+### Metadata catalogs
+
+Catalogs, in the context of Iceberg, refer to the central storage of metadata.
+Catalogs are also used to provide the atomic compare-and-swap needed to support 
+[serializable isolation in Iceberg](https://iceberg.apache.org/docs/latest/reliability).
+We'll refer to them as metadata catalogs to avoid confusion with Trino 
+[catalogs](https://trino.io/docs/current/sql/show-catalogs.html).
+
+The two existing catalogs supported in Trino's Iceberg connector is the 
+[Hive Metastore Service]({{site.url}}/docs/current/connector/iceberg.html#hive-metastore-catalog)
+and the AWS metastore counterpart of the Hive Metastore, Glue. While this 
+provides a nice migration from the Hive model, many are looking to replace these
```

Comment:
Do we want to mention that the HMS is barely used in Iceberg since most metadata is in the filesystem / tablformat itself?

Reply:
And hence the catalog can be much simpler than HMS

Reply:
@bitsondatadev ?

Reply:
Ah, I meant to come back to this one...I can make that mention yes.